tests/integration/security/authorization_test.go

```diff
@@ -87,34 +87,39 @@ func TestAuthorization_mTLS(t *testing.T) {
 				ctx.NewSubTest(fmt.Sprintf("From %s", cluster.StableName())).Run(func(ctx framework.TestContext) {
 					a := apps.A.Match(echo.InCluster(cluster).And(echo.Namespace(apps.Namespace1.Name())))
 					c := apps.C.Match(echo.InCluster(cluster).And(echo.Namespace(apps.Namespace2.Name())))
-					newTestCase := func(from echo.Instance, path string, expectAllowed bool) rbacUtil.TestCase {
+					b := apps.B.Match(echo.Namespace(apps.Namespace1.Name()))
```

Comment:
Are there multiple versions of `A`, `B`, and `C` in different namespaces? Or is every `a`/`b` in ns1 and `c` is always in ns2? 

You probably don't need the `Namespace` matcher.

Reply:
Yes, there are ns1(a, b, c, d, multiversion, naked, vm, headless, headlessnaked), ns2(b, c), and ns3(c). a is only in ns1, b is in ns1 and ns2, c is in ns1, ns2 and ns3.